src/clients/memberclient.cpp

```diff
@@ -267,6 +284,13 @@ int main(int argc, char** argv)
   auto ack =
     app.add_subcommand("ack", "Acknowledge self added into the network");
 
+  std::string nodes_file;
+  auto add_node = app.add_subcommand("add_node", "Make a node trusted");
```

Comment:
do nodes get added as trusted or is the comment wrong?

Reply:
That's a copy paste error, thanks for noticing :)